### DISCUSSION TITLE
2024 rules

### DISCUSSION THREAD

**Discussion by andrei-ghiura** [ebullient/ttrpg-convert-cli#554] [Ideas]
Hello, any news on when/if the new 2024 will be convertable? 

There was a new "freeRules2024" attribute introduced. Also there are a lot of reprinted items and duplicates and I am guessing the app doesn't really like the new changes 5etools made. There is a mirror with `-3` in the repo name that I am trying with.

**Comment by ebullient** [comment]
Duplicate of #548 and #532 and #537 

I will get there. One change in particular is significant.

**Reply by andrei-ghiura** [reply]
Thanks for the quick reply. Just wanted to make sure you didn't abandon this after the taking down of `-2` mirror. 

You're doing God's work, man. Thanks a lot!

**Reply by ebullient** [reply]
Not a man, but thanks all the same. ;)
